permission_handler_platform_interface/lib/src/method_channel/method_channel_permission_handler.dart

```diff
@@ -50,7 +50,7 @@ class MethodChannelPermissionHandler extends PermissionHandlerPlatform {
   ///
   /// Returns [true] if the app settings page could be opened, otherwise
   /// [false].
-  Future<bool> openAppSettings() async {
+  Future<bool?> openAppSettings() async {
     final wasOpened = await _methodChannel.invokeMethod('openAppSettings');
     return wasOpened;
   }
```

Comment:
I think this should be allowed to be `null`. The method should always return one of the following values:

- `true` if the App settings could be opened;
- `false` if the App settings could not be opened;
-  Or an exception if something went wrong.

A possible solution would be to enforce the returned value using the bank (!) operator (this will work since the native code will always return either `true` or `false`):

```suggestion
  Future<bool> openAppSettings() async {
    final wasOpened = await _methodChannel.invokeMethod('openAppSettings');
    return wasOpened!;
  }
```